bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -84,11 +87,19 @@ for trigger_file, injection_file in zip(args.trigger_files,
     if 'foreground/time1' in f:
         multi_ifo_style = False
         ifo_list = ['H1', 'L1']
+        assert args.min_required_ifos > 2
+            raise NotImplementedError('min-required-ifos must be less than or '
```

Comment:
Travis didn't like this (unexpected indent). I suppose what you want to do is if args.min_required_ifos > 2 then raise error. I think RuntimeError would be preferable since if the search is only 2-ifo it's logically impossible to require 3 ifos being on.  Also the error message should say the number of ifos in the search is 2 in this case.